presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1469,6 +1469,32 @@ public void testCreateAndInsert()
         assertEqualsIgnoreOrder(actualAfterTransaction, expected);
     }
 
+    @Test
+    public void testRenamePartitionColumn()
+            throws Exception
+    {
+        for (HiveStorageFormat storageFormat : HiveStorageFormat.values()) {
+            testRenamePartitionColumn(storageFormat);
+        }
+    }
+
+    private void testRenamePartitionColumn(HiveStorageFormat storageFormat)
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_partitioned_table (" +
+                "_bigint BIGINT" +
+                ", _partition_bigint BIGINT" +
+                ") " +
+                "WITH (" +
+                "format = '" + storageFormat + "', " +
+                "partitioned_by = ARRAY[ '_partition_bigint' ]" +
+                ") ";
+
+        assertUpdate(createTable);
+        assertQueryFails("ALTER TABLE test_partitioned_table RENAME COLUMN _partition_bigint TO new_name", "Renaming partition columns is not supported");
```

Comment:
I realize that we don't have a test for partition rename here at all. I would add one.

I would also verify that querying the table with the original column name continues to work.